en/08_Changelogs/5.4.0.md

```diff
@@ -203,6 +203,12 @@ The `SilverStripe\CMS\Model\SiteTree.DESCRIPTION` localisation key (along with t
 - [`RemoteFileModalExtension::getSchemaResponse()`](api:SilverStripe\AssetAdmin\Extensions\RemoteFileModalExtension::getSchemaResponse()) has been deprecated. It will be replaced with `RemoteFileModalExtension::getOwner()->getSchemaResponse()`.
 - [`InternalLinkModalExtension`](api:SilverStripe\CMS\Forms\InternalLinkModalExtension) has been deprecated. It will be replaced with configuration on [`ModalController`](api:SilverStripe\Admin\ModalController).
 - [`FormField::validate()`](api:SilverStripe\Forms\FormField::validate()) will take zero arguments and return a ValidationResult object instead of a boolean in CMS 6.0.0.
+- [`FieldsValidator`](api:SilverStripe\Forms\FieldsValidator) has been deprecated. It will be replaced with functionality inside Silverstripe\Forms\Form::validate().
```

Comment:
```suggestion
- [`FieldsValidator`](api:SilverStripe\Forms\FieldsValidator) has been deprecated. It will be replaced with functionality inside [`Form::validate()`](api:Silverstripe\Forms\Form::validate()).
```

Reply:
Updated